src/gorilla.h

```diff
@@ -0,0 +1,63 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#ifndef GORILLA_H
+#define GORILLA_H
+
+#include <sys/types.h>      // u_int_t
+#include <stdbool.h>        // bool
+
+#define timestamp_t u_int64_t
```

Comment:
Did you move timestamp_t to here or did you redfined it?

Reply:
redefined since neither of gorilla.c or gorilla.h inherits from const.h